js/main.js

```diff
@@ -71,6 +95,6 @@ addBtn.addEventListener('click', (e) => {
   document.getElementById('author').value = '';
 });
 
```

Comment:
**[OPTIONAL]**
- [ ] It would be nice if your data is preserved in localStorage which was a requirement in your previous milestone :+1: to make your project features complete :+1: